src/controllerbluetooth.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This bash file connects and disconnects the Xbox One S controller to the
+# Raspberry Pis depending on the argument passed using the controller's.
+# MAC address. If the argument passed is "connect" then the controller is
+# connected, if "disconnect" is passed then the controller is disconnected
+
+option=$1
+
+if [[ $option == "connect" ]]; then
+    echo 'connect 5C:BA:37:26:6D:9A' | sudo bluetoothctl
```

Comment:
Instead of hard coding the BTADDR, can we replace it with a command-line argument? That way the script can be called by just saying something like:

./script.sh connect 5C:BA:37:26:6D:9A